bindata/network/ovn-kubernetes/ovnkube-master.yaml

```diff
@@ -67,6 +71,32 @@ spec:
             source /env/_master
             set +o allexport
           fi
+          
+          # Perform DNS resolution against the DNS resolver first and /etc/hosts second
+          sed -i 's/hosts:      files dns myhostname/hosts:      dns files myhostname/g' /etc/nsswitch.conf
```

Comment:
is this change needed before the `resolv.conf` switch? (ie, could this be done at the end of `watch_core_dns_ep`?)

Reply:
Indeed, that will speed things up during the first phase